api/src/db/DynamoBusinessesDataClient.test.ts

```diff
@@ -95,11 +84,7 @@ describe("DynamoBusinessesDataClient", () => {
     await dynamoBusinessesDataClient.put(businessData);
     const expectedValue = await dynamoBusinessesDataClient.findAllByIndustry(industry);
     expect(expectedValue[0]).toBeDefined();
```

Comment:
sand - does it make sense to have an assertion where expectedValue has a length of 1?

Reply:
The key difference here is that `findAllByIndustry`returns an array of objects, whereas `findByEncryptedTaxId` returns a single object or undefined. In this test, I'll assert based on the array's length using .toHaveLength(1) to verify that exactly one business is returned. After that, I'll assert the value of the first item in the array, ensuring it matches the businessData. This approach is consistent with the previous test where we assert based on length first and then validate the actual object